:rotating_light: NIGHTLY DEPENDENCIES TEST: failure on `refs/heads/main`

The run `12928889796` of the workflow testing Nilearn with the nightly build of its dependencies failed on `refs/heads/main`.

You can view the report here: https://github.com/nilearn/nilearn/actions/runs/12928889796

Pinging @nilearn/core

```
  FAILED nilearn/plotting/tests/test_surf_plotting.py::test_plotly_savefig - choreographer.browser.BrowserFailedError: Close() was called before the browser finished opening- maybe it crashed?
  FAILED nilearn/plotting/tests/test_surf_plotting.py::test_plot_surf[plotly] - choreographer.browser.BrowserFailedError: Close() was called before the browser finished opening- maybe it crashed?
```

eventhough the test only fails for the pre-release versions of our dependencies I have not been able to spot differences in the version of the dependencies

--- a/tox.ini
+++ b/tox.ini
@@ -43,7 +43,7 @@ deps =
     {[matplotlib]deps}
     kaleido
     plotly
-    kaleido ; platform_system != 'Windows'
+    kaleido==0.2.1 ; platform_system != 'Windows'
     kaleido==0.1.0.post1 ; platform_system == 'Windows'
 
 [global_var]

